### ISSUE TITLE
Cursor not shown in Math Preview Panel

### ISSUE THREAD

**Issue by lukas-lang** [James-Yu/LaTeX-Workshop#3212]
### Disable all the other extensions except for LaTeX Workshop, restart VS Code, and check that you still see this issue. [Required]

You still see this issue?: Yes

### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.

You visited the wiki?: Yes

### If your issue is with compiling a document (not having to do with finding the root file of a project), check first that you can compile manually.

You can compile a TeX document manually?: not applicable

## Describe the bug [Required]

Even though I have `"latex-workshop.hover.preview.cursor.enabled": true`, the cursor is not visible in the Math Preview Panel. It is however visible in the hover preview, as expected.

### To Reproduce

* Type some equation, e.g. `\(f(x)=x^2\)`
* Open the Math Preview Panel
* Position the cursor inside the equation, and hover over it.
* Note how the cursor is only rendered in the hover preview:
![image](https://user-images.githubusercontent.com/33105866/159298660-43fc9423-e8a4-42fc-865b-de1feaf89651.png)

### Expected behavior

I would have expected the cursor to be rendered in the preview panel.

## Logs [Required]

Please paste the whole log messages here, not parts of ones. The log should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

### LaTeX Workshop Output [Required]

```
[16:49:27] Initializing LaTeX Workshop.
[16:49:27] Extension root: c:\Users\lukas\.vscode\extensions\james-yu.latex-workshop-8.23.0
[16:49:27] $PATH: C:\Program Files\Python38\Scripts\;C:\Program Files\Python38\;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\ProgramData\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\SysWOW64\WindowsPowerShell\v1.0\Modules\TShell\TShell\;C:\Program Files\Java\jdk1.8.0_101\\bin;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Users\lukas\AppData\Local\Microsoft\WindowsApps;C:\Users\lukas\AppData\Local\Android\android-sdk\platform-tools;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Wolfram Research\WolframScript\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\nodejs\;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\Program Files\IVI Foundation\VISA\Win64\Bin\;C:\Program Files (x86)\IVI Foundation\VISA\WinNT\Bin\;C:\Program Files\Wolfram Research\WolframScript\;C:\Program Files\MATLAB\R2021a\bin;C:\Program Files\Git\cmd;C:\Program Files\dotnet\;C:\Users\lukas\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Microsoft VS Code\bin;C:\Users\lukas\AppData\Roaming\npm;C:\Users\lukas\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\lukas\AppData\Local\Programs\MiKTeX 2.9\miktex\bin\x64\;C:\Users\lukas\AppData\Local\Programs\Fiddler;C:\Users\lukas\Programme\CommandLineTools;;C:\Users\lukas\AppData\Roaming\Mathematica\ffmpeg-4.4-full_build-shared\bin\
[16:49:27] $SHELL: undefined
[16:49:27] vscode.env.appName: Visual Studio Code
[16:49:27] vscode.env.remoteName: undefined
[16:49:27] vscode.env.uiKind: 1
[16:49:27] editor.acceptSuggestionOnEnter: "on"
[16:49:27] latex-workshop.bind.enter.key: true
[16:49:27] latex-workshop.docker.enabled: false
[16:49:27] latex-workshop.docker.image.latex: ""
[16:49:27] latex-workshop.hover.preview.mathjax.extensions: [
 "physics",
 "cancel",
 "mathtools"
]
[16:49:27] latex-workshop.intellisense.package.enabled: true
[16:49:27] latex-workshop.intellisense.update.aggressive.enabled: false
[16:49:27] latex-workshop.intellisense.update.delay: 1000
[16:49:27] latex-workshop.latex.autoBuild.run: "onSave"
[16:49:27] latex-workshop.latex.outDir: "%DIR%"
[16:49:27] latex-workshop.latex.recipes: [
 {
  "name": "latexmk 🔃",
  "tools": [
   "latexmk"
  ]
 },
 {
  "name": "latexmk (latexmkrc)",
  "tools": [
   "latexmk_rconly"
  ]
 },
 {
  "name": "latexmk (lualatex)",
  "tools": [
   "lualatexmk"
  ]
 },
 {
  "name": "pdflatex ➞ bibtex ➞ pdflatex × 2",
  "tools": [
   "pdflatex",
   "bibtex",
   "pdflatex",
   "pdflatex"
  ]
 },
 {
  "name": "Compile Rnw files",
  "tools": [
   "rnw2tex",
   "latexmk"
  ]
 },
 {
  "name": "Compile Jnw files",
  "tools": [
   "jnw2tex",
   "latexmk"
  ]
 },
 {
  "name": "tectonic",
  "tools": [
   "tectonic"
  ]
 }
]
[16:49:27] latex-workshop.latex.tools: [
 {
  "name": "latexmk",
  "command": "latexmk",
  "args": [
   "--shell-escape",
   "-synctex=1",
   "-interaction=nonstopmode",
   "-file-line-error",
   "-pdf",
   "-outdir=%OUTDIR%",
   "%DOC%"
  ],
  "env": {}
 },
 {
  "name": "pdflatex",
  "command": "pdflatex",
  "args": [
   "--shell-escape",
   "-synctex=1",
   "-interaction=nonstopmode",
   "-file-line-error",
   "%DOC%"
  ],
  "env": {}
 },
 {
  "name": "bibtex",
  "command": "bibtex",
  "args": [
   "%DOCFILE%"
  ],
  "env": {}
 }
]
[16:49:27] latex-workshop.viewer.pdf.internal.keyboardEvent: "auto"
[16:49:27] Creating a new file watcher.
[16:49:27] watcherOptions: {"useFsEvents":false,"usePolling":false,"interval":300,"binaryInterval":1000,"awaitWriteFinish":{"stabilityThreshold":250}}
[16:49:27] Creating PDF file watcher.
[16:49:27] watcherOptions: {"useFsEvents":false,"usePolling":false,"interval":300,"binaryInterval":1000,"awaitWriteFinish":{}}
[16:49:27] Creating Bib file watcher.
[16:49:27] watcherOptions: {"useFsEvents":false,"usePolling":false,"interval":300,"binaryInterval":1000,"awaitWriteFinish":{"stabilityThreshold":250}}
[16:49:27] Set $LATEXWORKSHOP_DOCKER_LATEX: ""
[16:49:28] pdflatex is provided by MiKTeX
[16:49:28] [Server] Creating LaTeX Workshop http and websocket server.
[16:49:28] Bibtex format config: {"tab":"  ","case":"lowercase","left":"{","right":"}","trailingComma":false,"sort":["key"],"alignOnEqual":true,"sortFields":false,"fieldsOrder":[],"firstEntries":["string","xdata"]}
[16:49:28] LaTeX Workshop initialized.
[16:49:28] Bibtex format config: {"tab":"  ","case":"lowercase","left":"{","right":"}","trailingComma":false,"sort":["key"],"alignOnEqual":true,"sortFields":false,"fieldsOrder":[],"firstEntries":["string","xdata"]}
[16:49:28] Trigger characters for intellisense of LaTeX documents: ["\\",",","{"]
[16:49:28] Bibtex format config: {"tab":"  ","case":"lowercase","left":"{","right":"}","trailingComma":false,"sort":["key"],"alignOnEqual":true,"sortFields":false,"fieldsOrder":[],"firstEntries":["string","xdata"]}
[16:49:28] Current workspace folders: undefined
[16:49:28] Current workspaceRootDir: 
[16:49:28] The active document cannot be used as the root file: untitled:Untitled-1
[16:49:28] Linter for root file started.
[16:49:28] No root file found for linting.
[16:49:28] [Server] Server successfully started: {"address":"127.0.0.1","family":"IPv4","port":54735}
[16:49:28] Snippet data loaded.
[16:49:34] Math preview panel: opened
[16:49:34] Math preview panel: initialized

```

### Developer Tools Console [Required]

- Nothing relevant

## Screenshots

![image](https://user-images.githubusercontent.com/33105866/159299461-0d8e3890-27a9-4839-b6d4-cb9c77c8d080.png)

## Desktop [Required]

Please write exact version numbers. Please don't write `latest` instead of exact numbers.

 - OS: Windows 10, 21H1
 - VS Code version: 1.65.2
 - Extension version: 8.23.0
 - TeX distribution version: MiKTeX 22.1

### Are you using VSCodium?

No

### Are you using the Snap or Flatpack versions of VS Code?

No

### Are you using LaTeX Workshop with VS Code Remote?

No

### Additional context

Looking over the code, it like the cursor rendering is not implemented in the function called by the Preview Panel:

https://github.com/James-Yu/LaTeX-Workshop/blob/8c0b0c65effa3fcfe71acabca76c9690570e643f/src/providers/preview/mathpreview.ts#L107

Compare with the hover preview:

https://github.com/James-Yu/LaTeX-Workshop/blob/8c0b0c65effa3fcfe71acabca76c9690570e643f/src/providers/preview/mathpreview.ts#L51-L52

**Comment by jlelong** [comment]
You can try to set `latex-workshop.mathpreviewpanel.cursor.enabled` to true but note that this is an **experimental** feature

https://github.com/James-Yu/LaTeX-Workshop/blob/1bc44b9be228bd0c6f2875ac1668927834320f6e/package.json#L2268-L2273

**Comment by lukas-lang** [comment]
Thank you for the quick reply and suggestion!

I have tried to enable the setting by including `"latex-workshop.mathpreviewpanel.cursor.enabled": true` in my `settings.json` file, but without any luck: The setting is greyed out with the warning "Unknown Configuration Setting", and the cursor is still missing from the preview panel.
Do I need to configure anything else to use experimental features?

**Comment by jlelong** [comment]
> I have tried to enable the setting by including `"latex-workshop.mathpreviewpanel.cursor.enabled": true` in my `settings.json` file, but without any luck: The setting is greyed out with the warning "Unknown Configuration Setting", and the cursor is still missing from the preview panel.

This is because there is a typo in this setting in release `8.23.0`. Until a new release is available, you can use `latex-workshop.mathpreviewpanel.curosr.enabled`.

**Comment by tamuratak** [comment]
@jlelong  the feature was implemented after the release of `8.23.0` c9ab8ca7209f1e38b126915fa64120db68ef5ca3
